test/acceptance/test/utils.go

```diff
@@ -624,3 +605,85 @@ func takeScreenshot() string {
 	}
 	return ""
 }
+
+func getWaitTimeFromErr(errOutput string) (time.Duration, error) {
+	var re = regexp.MustCompile(`(?m)\[rate reset in (.*)\]`)
+	match := re.FindAllStringSubmatch(errOutput, -1)
+
+	if len(match) >= 1 && len(match[1][0]) > 0 {
+		duration, err := time.ParseDuration(match[1][0])
+		if err != nil {
+			return 0, fmt.Errorf("error pasing rate reset time %w", err)
+		}
+		return duration, nil
+	}
+
+	return 0, fmt.Errorf("could not found a rate reset on string: %s", errOutput)
+}
+
+func createRepository(repoName string, private bool) error {
+
+	visibility := gitprovider.RepositoryVisibilityPrivate
+	if !private {
```

Comment:
💡 positives are always easier to read.  Why not reverse this logic and have if the if statement just be `if private {}`?